### PR TITLE
fix shaded dependencies and add auto deploy by release

### PR DESCRIPTION
**Pull request by MarinaRazumovsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This appender uses [BigQueue](https://github.com/bulldog2011/bigqueue) implement
 <dependency>
     <groupId>io.logz.sender</groupId>
     <artifactId>logzio-sender</artifactId>
-    <version>1.0.7</version>
+    <version>1.0.8</version>
```
Comment (asafm) [inline README.md]: Let use revision number property?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+        - gpg --fast-import release/codesigning.asc
+        - rm release/codesigning.asc
+      deploy:
+        - provider: script
+          skip_cleanup: true
+          script: bash ./mvnw deploy -Drevision=$TRAVIS_TAG  --settings release/settings.xml
+          on:
+            tags: true
+            branch: master
+      after_deploy: echo 'Finish to deploy'
```
Comment (asafm) [inline .travis.yml]: `Deployed finished successfully`

File: logzio-sender/pom.xml [hunk 3]
```diff
@@ -103,7 +103,7 @@
         <dependency>
             <groupId>io.logz.sender</groupId>
             <artifactId>logzio-sender-test</artifactId>
-            <version>1.0.7</version>
+            <version>1.0.8</version>
```
Comment (asafm) [inline logzio-sender/pom.xml]: ${revision}?

File: .travis.yml (hunk 2 above)
Comment (MarinaRazumovsky) [reply .travis.yml]: 👍

**Pull request merged**